BraveWallet/Crypto/Stores/SwapTokenStore.swift

```diff
@@ -677,10 +684,10 @@ extension SwapTokenStore: BraveWalletKeyringServiceObserver {
   public func autoLockMinutesChanged() {
   }
   
-  public func selectedAccountChanged() {
+  public func selectedAccountChanged(_ coinType: BraveWallet.CoinType) {
     keyringService.defaultKeyringInfo { [self] keyringInfo in
       if !keyringInfo.accountInfos.isEmpty {
-        keyringService.selectedAccount { accountAddress in
+        keyringService.selectedAccount(.eth) { accountAddress in
```

Comment:
This should be `coinType` passed from arg

Reply:
Fixed at https://github.com/brave/brave-ios/pull/5029/commits/8a27b9cda073a4d5704076c0f3d5729708491888